capitalone/rubicon-ml
# 178 Make Pipeline, remove **kwargs from pipeline constructor

closes: #182 #178

---

## What
  * Added `make_pipeline` method to support constructing unnamed sklearn pipelines 
  * Changed signature for the `pipeline` to  `def __init__( self, project,steps,user_defined_loggers={},experiment_kwargs{"name": "RubiconPipeline experiment"},memory=None,verbose=False):`

## How to Test
  * ensure that all the tests still run with `pytest`
  * ensure that the sklearn integration notebook still runs without errors
